draft-ietf-httpbis-http2.xml

```diff
@@ -1904,12 +1906,13 @@ HTTP2-Settings    = token68
         </t>
 
         <t>
-          The RST_STREAM frame fully terminates the referenced stream and causes it to enter the
-          closed state. After receiving a RST_STREAM on a stream, the receiver MUST NOT send
-          additional frames for that stream, with the exception of <x:ref>PRIORITY</x:ref>. However,
-          after sending the RST_STREAM, the sending endpoint MUST be prepared to receive and process
-          additional frames sent on the stream that might have been sent by the peer prior to the
-          arrival of the RST_STREAM.
+          Receiving a RST_STREAM frame fully terminates the stream and causes it to enter the 
+          "closed" state.  After receiving a RST_STREAM on a stream, the receiver MUST NOT send
+          additional frames for that stream, with the exception of <x:ref>PRIORITY</x:ref>. 
+          Sending a RST_STREAM causes the stream to enter the "reset" state and wait for a RST_STREAM
+          in response. After sending the RST_STREAM, the sending endpoint MUST be prepared to receive 

```

Comment:
Humm not ideal in general but the variety of terminology for this type of situation seems to be:
1. `in response`
2. `remote peer`
3. some form of `endpoint`

I'm not a big fan of the `response` terminology because it has meaning within the HTTP protocol.